Evaluation/Input/Content/Section2.2_Data.md

```diff
@@ -65,40 +65,40 @@ A literature search was performed to collect available information on physical c
+|       |             | fed        | 100            | SD     | Itraconazole                                                 | mbe                  | [Van Peer 1989](#5-References)     |
+|       |             |            |                | OD     | Itraconazole                                                 | mbe                  | [Van Peer 1989](#5-References)    |
+|       |             |            |                |        | Itraconazole                                                 | mv                   | [Hardin 1988](#5-References)       |
+|       |             |            | 200            | SD     | Itraconazole<br />Hydroxy-Itr.                               | mbe                  | [Barone 1993](#5-References)       |
+|       |             |            |                |        | Itraconazole<br />Hydroxy-Itr.                               | mbe                  | [Barone 1998b](#5-References)      |
 |       |             |            |                |        | Itraconazole                                                 | mv                   | [Neuvonen 1996](#5-References)      |
-|       |             |            | 200            | OD     | Itraconazole                                                 | mv                   | [Hardin  1988](#5-References)       |
-|       |             |            |                | BID    | Itraconazole                                                 | mbe                  | [Barone  1993](#5-References)       |
-|       |             |            |                |        | Itraconazole<br />Hydroxy-Itr.                               | mv                   | [Hardin  1988](#5-References)       |
+|       |             |            | 200            | OD     | Itraconazole                                                 | mv                   | [Hardin 1988](#5-References)       |
+|       |             |            |                | BID    | Itraconazole                                                 | mbe                  | [Barone 1993](#5-References)       |
+|       |             |            |                |        | Itraconazole<br />Hydroxy-Itr.                               | mv                   | [Hardin 1988](#5-References)       |
 
-<sup>†</sup> *mbb* model building basic: used to inform the basic model parameters (see [Section 2.3.4](#234-Automated-Parameter-Identification)); *mbe* model building extended: used to inform solubility (and formulation-depenendent) parameter only (see [Section 2.3.4](#234-Automated-Parameter-Identification)); *mv* model verification only
+<sup>†</sup> *mbb* model building basic: used to inform the basic model parameters (see [Section 2.3.5](#235-Automated-Parameter-Identification)); *mbe* model building extended: used to inform solubility and, if applicable, formulation-depenendent parameters only (see [Section 2.3.5](#235-Automated-Parameter-Identification)); *mv* model verification only
```

Comment:
why is all this compound data part of a content md?
It can be generated by the qualification runner during a qualification run (Inputs section of a Quali plan).

Reply:
Well, inputs justs lists the parameter valus but not the modelling strategy